assets/js/src/core/components/switch/switch.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React from 'react'
+import type { SwitchProps as AntdSwitchProps } from 'antd/es/switch/index'
+import { Switch as AntdSwitch } from 'antd'
+import { Flex } from '@Pimcore/components/flex/flex'
+
+export interface SwitchProps extends AntdSwitchProps {
+  tag: string
```

Comment:
I would rather call it a label. When I read tag, I have to think of our tag component.

Have you checked if this is maybe also already possible with the Form/FormItem component from ant? 
It would be good to have a story example for that to. Especially with the placement of the label.

Reply:
Defining where the label sits via a form component is not possible but I have created a wrapper and added a story for it. 
Currently the wrapper does not do a whole lot though . Let me know what you think.